src/status_im2/contexts/quo_preview/wallet/account_origin.cljs

```diff
@@ -0,0 +1,53 @@
+(ns status-im2.contexts.quo-preview.wallet.account-origin
+  (:require [quo2.core :as quo]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [status-im2.common.resources :as resources]))
+
+(def descriptor
+  [{:label   "Type:"
+    :type    :select
+    :key     :type
+    :options [{:key   :default-keypair
+               :value "Default Keypair"}
```

Comment:
You don't need to explicitly pass `:value` if you think the humanized text deduced from `:default-keypair` is good enough. In this case it is, `:default-keypair` will be automatically transformed to `"Default keypair"`. You can also remove redundant `:label` values.

For reference: here's the function that does this work for us https://github.com/status-im/status-mobile/blob/b0f748cb739f1de81ebc775e0cc77e4e2de85ef1/src/status_im2/contexts/quo_preview/preview.cljs#L21

This is how the descriptor should look like:

```clojure
(def descriptor
  [{:type    :select
    :key     :type
    :options [{:key :default-keypair}
              {:key :recovery-phrase}
              {:key :private-key}]}
   {:type    :select
    :key     :stored
    :options [{:key :on-device}
              {:key :on-keycard}]}])
```

Reply:
Very smaller and cleaner. Done